### DISCUSSION TITLE
When should we bump our Node LTS requirement

### DISCUSSION THREAD

**Discussion by fgreinacher** [semantic-release/semantic-release#3088] [General] [answered]
Today I saw the first dependency that needs at least Node 20 (new LTS): https://github.com/semantic-release/gitlab/pull/639

I know the Node version policy, was just wondering if and when we should drop Node 18 (previous LTS) support.

**Comment by travi** [comment]
probably soon. also just ran into this, which slipped past our current test suite: https://github.com/semantic-release/semantic-release/issues/3089.

since the gitlab plugin is not a dependency of core, i would also say that i think it is ok for it to drop node v18 support ahead of core if anything does delay us getting a new major of core out in a reasonable timeframe.

**Reply by travi** [reply]
while not a prod dependency, [ava just raised its supported node version minimums](https://github.com/semantic-release/semantic-release/pull/3087), so i wouldnt be surprised if others follow suit. because of that, i suggest we set a minimum of v20.8 when we do raise ours.

**Reply by travi** [reply, marked as answer]
started a PR for the next major: #3111
